src/google/protobuf/compiler/objectivec/objectivec_enum.cc

```diff
@@ -41,6 +41,17 @@ namespace google {
 namespace protobuf {
 namespace compiler {
 namespace objectivec {
+namespace {
+std::string SafelyPrintIntToCode(int v) {
+  if (v == std::numeric_limits<int>::min()) {
+    // Some compilers try to parse -2147483648 as two tokens and then get spicy
+    // about the fact that +2147483648 cannot be represented as an int.
+    return "-2147483647 - 1";
+  } else {
+    return absl::StrCat(v);
```

Comment:
Oops, I don't think you want `absl::` on this.

Reply:
yeah, just removed it